{"uuid": "2dc5e4b9-16b0-471d-b8e7-d3f2593a6407", "children": ["9d3c555b-5892-49fa-9d94-7a6f09dcd795"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1696848891639, "stop": 1696848891639}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\fardi\\PycharmProjects\\Rokomari_WebAutomation_Project\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\fardi\\PycharmProjects\\Rokomari_WebAutomation_Project\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\fardi\\PycharmProjects\\Rokomari_WebAutomation_Project\\TestCases\\conftest.py\", line 30, in log_on_failure\n    if item.rep_call.failed:\n"}, "start": 1696848897276, "stop": 1696848897276}], "start": 1696848891639, "stop": 1696848897276}